elections/steering/2022/README.md

```diff
@@ -128,19 +128,22 @@ Examples of contributions that would NOT be considered:
   public announcement is a weekend.
 -->
 
-| Date    | Event                                                                 |
-|:--------|:----------------------------------------------------------------------|
-| July 22 | Steering Committee selects Election Committee                         |
-|         | Announcement of Election and publication of voters.md                 |
-|         | Steering Committee Meeting with Q+A with the candidates and community |
-|         | All candidate bios due by 0000 UTC (5pm PST)                          |
-|         | Election Begins via email ballots                                     |
-|         | Deadline to submit voter exception requests                           |
-|         | Election Closes by 0000 UTC (5pm PST)                                 |
-|         | Private announcement of Results to SC members not up for election     |
-|         | Private announcement of Results to all candidates                     |
-|         | Public announcement of Results at Public Steering Committee Meeting   |
-|         | Election Retro                                                        |
+| Date                    | Event                                                                 |
+|:------------------------|:----------------------------------------------------------------------|
+| Friday, July 22         | Steering Committee selects Election Committee                         |
+| Monday, August 15       | Announcement of Election and publication of voters.md                 |
+| ???, August ??          | Steering Committee Q+A for the candidates                             |
```

Comment:
this should probably go after the nominations deadline, so the candidates are known, right?

Reply:
I think we've previously done this prior to the deadline so that folks can ask questions to see if they actually want to run, and understand more about the job.

Reply:
ah, ok... I couldn't remember if it was an open invitation or if candidates were specifically invited

Reply:
if the Q+A will be open to potential candidates, keeping this ahead of nomination close seems fine, so feel free to resolve this comment